main.js

```diff
@@ -0,0 +1,152 @@
+const BASE_URL = "https://45220a845ac73b32.mokky.dev";
+
+const input = document.getElementById("input");
+const form = document.getElementById("form");
+const div = document.getElementById("div");
+
+form.addEventListener("click", (e) => {
+  e.preventDefault();
+  const todoValue = input.value.trim();
+  if (todoValue!=="") {
+    const newTodo = {
+      id: Date.now().toString(),
+      title: todoValue,
+      completed: false,
+    };
+    input.value = "";
+    postTodo(newTodo);
+  }else{alert("Can't be empty")}
+});
+
+const postTodo = async (objectTodo) => {
+  try {
+    const response = await fetch(`${BASE_URL}/users`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(objectTodo),
+    });
+    const data = await response.json();
+    console.log(data);
+    getTodos();
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const getTodos = async () => {
+  try {
+    const response = await fetch(`${BASE_URL}/users `);
+    const data = await response.json();
+
+    console.log();
+    renderTodo(data);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const renderTodo = (newArray) => {
```

Comment:
const renderTodo = (newArray = []) => {
  div.innerHTML = "";
  newArray.map((todo) => {
    console.log(todo);

    const pTag = document.createElement("p");
    pTag.textContent = todo.title;
    pTag.className = "text";

    const deleteButton = document.createElement("button");
    deleteButton.textContent = "Delete";
    deleteButton.className = "buttunDe";

    const updateButton = document.createElement("button");
    updateButton.textContent = "Update";
    updateButton.className = "buttonUp";
    const checkbox = document.createElement("input");
    checkbox.type = "checkbox";
    checkbox.className = "check";
    checkbox.checked = todo.completed;

    deleteButton.addEventListener("click", () => {
      if (confirm("Do you want to delet?")) {
        deleteTodo(todo.id);
        localStorage.removeItem("obj");
      } else {
        return false;
      }
    });
    updateButton.addEventListener("click", () =>
      updateTodo(todo.id, newArray)
    );

    if (todo.completed) {
      pTag.style.textDecoration = "line-through";
      pTag.style.textDecorationColor = "red";
    } else {
      pTag.style.textDecoration = "none";
    }
    checkbox.addEventListener("change", () => {
      completedTodo(todo.id, todo.completed);
    });
    const box = document.createElement("div");

    box.append(deleteButton, updateButton, checkbox);
    pTag.appendChild(box);
    div.appendChild(pTag);
  });
};